### PR TITLE
About page

### PR DESCRIPTION
**Pull request by r0ddy**
This PR does the following:
- adds JQuery to clone the navbar onto each html page
- adds an about page with content

### CODE REVIEW

File: portfolio/src/main/webapp/nav.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <a href="about.html" class="navLink">
+        <h4 class="navText">About</h3>
+    </a>
+    <a href="" class="navLink">
+        <h4 class="navText">Projects</h3>
+    </a>
+    <a href="" class="navLink">
+        <h4 class="navText">Blog</h3>
+    </a>
+</div>
```
Comment (RodrigoNieves) [inline portfolio/src/main/webapp/nav.html:17]: please add a new line at the end of the file

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,17 +12,6 @@
...
-  // Pick a random greeting.
-  const quote = quotes[Math.floor(Math.random() * quotes.length)];
-
-  // Add it to the page.
-  const quoteContainer = document.getElementById('quote-container');
-  quoteContainer.innerText = quote;
-}
+$(function(){
+    $("nav").load("nav.html");
+});
```
Comment (RodrigoNieves) [inline portfolio/src/main/webapp/script.js:17]: pleas add a new line at the end of the file

File: portfolio/src/main/webapp/nav.html (hunk 1 above)
Comment (r0ddy) [reply portfolio/src/main/webapp/nav.html:17]: Will do, but is it because of style or is it more semantic?

Comment (RodrigoNieves) [reply portfolio/src/main/webapp/nav.html:17]: It is just style.

**Pull request merged**